### ISSUE TITLE
Ignore case in autocomplete

### ISSUE THREAD

**Issue by Lucasmiguelmac** [arabdevelop/svelte-formly#43]
Autocomplete field is awesome, but as of today you can't miss the case, otherwise it won't autocomplete

**Comment by kamalkech** [comment]
can you give more details to answer u

**Comment by Lucasmiguelmac** [comment]
Yes. If in your autocomplete options you have `"Spain"` as an item. If you type "spain", it won't autocomplete to spain because the latter one starts with a lowercase "s" instead of an uppercase one ("S").

We could implement this with a `ignoreCase` boolean.

**Comment by dyaliCode** [comment]
@Lucasmiguelmac i guess this issue it s already fixed on last version with big update like add typescript...etc, checkout new documentation site [https://svelte-formly-ts-documentation.netlify.app](https://svelte-formly-ts-documentation.netlify.app)
